registry/storage/signedmanifesthandler.go

```diff
@@ -36,6 +43,14 @@ func (ms *signedManifestHandler) Unmarshal(ctx context.Context, dgst digest.Dige
 		return nil, err
 	}
 
+	if ms.repository.schema1SigningKey != nil {
+		if err := jsig.Sign(ms.repository.schema1SigningKey); err != nil {
+			return nil, err
+		}
+	} else if !ms.repository.schema1SignaturesEnabled {
+		return nil, fmt.Errorf("Missing signing key with signature store disabled")
```

Comment:
I am pretty sure we have been strict on the Go coding guidelines for awhile, any cases are likely very old or just slipped through the cracks. Do we not have the linter configured to a level which enforces this?